### DISCUSSION TITLE
Duplicação de páginas no Google

### DISCUSSION THREAD

**Discussion by mahenrique94** [frontendbr/forum#983] [Perguntas e Respostas]
Fala ai galera, beleza?

Uma dúvida: Recentemente estou investindo mais tempo e dedicação no meu blog, também escrevo para blogs de terceiros, mas, eu simplesmente replico os posts feitos no meu blog para esses outros e referencio o meu como fonte original.

Até então, lendo a documentação do Google, eu vi que dessa maneira não tem problemas de SEO para mim e para esses outros, até porque o próprio Medium faz essa importação e referência.

Mas a PA (Page Authority) e DA (Domain Authority) do blog não está aumentando muito, então fiquei na dúvida se o problema poderia ser essa duplicação de páginas ou demora para subir assim mesmo.

Abraçooos

**Comment by kvnol** [comment]
Faz muito tempo que não trabalho diretamente com SEO, mas uns 3/4 anos atrás era ruim ter esse conteúdo duplicado em diferentes endereços.

**Comment by tarciozemel** [comment]
Conteúdo duplicado (provavelmente) não é penalizado; conteúdo copiado, sim.

https://www.hobo-web.co.uk/duplicate-content-problems/

**Comment by mahenrique94** [comment]
O que eu li na documentação do próprio Google é que se tiver a referência para a fonte original, não tem problema.
